docs/streams/upgrade-guide.html

```diff
@@ -86,7 +86,11 @@ <h3><a id="streams_api_changes_120" href="#streams_api_changes_120">Streams API
         to let users specify inner serdes if the default serde classes are windowed serdes.
         For more details, see <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-265%3A+Make+Windowed+Serde+to+public+APIs">KIP-265</a>.
     /<p>
-    
+
+     <p>
+	We have deprecated <code>StreamsConfig</code> in <code>KafkaStreams</code> constructors. Now we only take in <code>java.util.Properties</code> since <code>StreamsConfig</code> is immutable and is created from a Properties object itself.
```

Comment:
Nit: avoid "tabs" and use whitespace only.

Sounds a little bit as if `StreamsConfig` was deprecates. What about:
```
We deprecated all constructors of <code>KafkaStreams</code> that take a <code>StreamsConfig</code> as parameter.
Please use the corresponding constructors that accept <code>java.util.Properties</code> instead.
For more details, see <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-245%3A+Use+Properties+instead+of+StreamsConfig+in+KafkaStreams+constructor">KIP-245</a>.
```

I don't think we need to explain why we changed it (people can read the KIP if they are interested).